cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/AbstractConductingEquipmentConversion.java

```diff
@@ -244,7 +244,6 @@ public void convertToDanglingLine(int boundarySide, double r, double x, double g
         }
         context.terminalMapping().add(terminalId(boundarySide), dl.getBoundary(), 2);
         dl.addAlias(terminalId(boundarySide), Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + "Terminal_Boundary");
-        dl.setProperty(Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + dl.getId() + ".Terminal_Boundary", terminalId(boundarySide)); // TODO: delete when aliases are correctly handled by mergedlines
```

Comment:
This can't go, the merging view **does not** handle aliases correctly, it is the only way to keep the boundaries in merging view's tie lines.

Reply:
kept the property